pkg/types/contract_reader.go

```diff
@@ -93,9 +93,9 @@ func (brr *BatchReadResult) SetResult(returnValue any, err error) {
 }
 
 type Head struct {
-	Identifier string
-	Hash       []byte
-	Timestamp  uint64
+	Height    string
+	Hash      []byte
+	Timestamp uint64
```

Comment:
Are unix timestamps with millisecond precision be sufficiently general? Or do we need more precision? (I suppose we could just add a field later for nanos or something :thinking:)

Reply:
We can add more precision if needed later, not sure if all chains even more precision?

Reply:
I always find Unix the most universal, what advantage do you see in using protobuf seconds?